.circleci/config.yml

```diff
@@ -38,5 +38,11 @@ jobs:
             - ~/.m2
           key: v1-dependencies-{{ checksum "project.clj" }}
 
+      # install planck
+      - run: wget https://planck-repl.org/releases/2.25.0-debian-9.0-x86_64/planck && chmod a+x planck &&  sudo mv planck /usr/local/bin
+      - run: sudo apt-get install clojure
```

Comment:
Probably last question/remark: Is this installation of Clojure required? If so, why?